Ginger/Ginger/WindowExplorer/Common/ControlActionsPage.xaml.cs

```diff
@@ -182,7 +182,9 @@ private void AddActionButton_Click(object sender, RoutedEventArgs e)
                 actUI.GetOrCreateInputParam(ActUIElement.Fields.WhereColumnTitle, aaa.GetInputParamValue(ActUIElement.Fields.WhereColumnTitle));
                 actUI.GetOrCreateInputParam(ActUIElement.Fields.WhereColumnValue, aaa.GetInputParamValue(ActUIElement.Fields.WhereColumnValue));
                 actUI.GetOrCreateInputParam(ActUIElement.Fields.WhereOperator, aaa.GetInputParamValue(ActUIElement.Fields.WhereOperator));
-                actUI.GetOrCreateInputParam(ActUIElement.Fields.WhereProperty, aaa.GetInputParamValue(ActUIElement.Fields.WhereProperty));
+                actUI.GetOrCreateInputParam(ActUIElement.Fields.WhereProperty, aaa.GetInputParamValue(ActUIElement.Fields.WhereProperty));               
+                actUI.GetOrCreateInputParam(ActUIElement.Fields.XCoordinate, mElementInfo.GetInputParamValue(ActUIElement.Fields.XCoordinate));
```

Comment:
This is not the correct way.  We need 1 function which will return all the special values and not to read one by one.
We need to get rid of this code which is reading properties one by one. Tomorrow if new field is added we need to add code for it